activerecord-jdbc-adapter.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |gem|
   gem.authors = ['Nick Sieger, Ola Bini, Karol Bucek and JRuby contributors']
   gem.email = ['nick@nicksieger.com', 'ola.bini@gmail.com', 'self@kares.org']
   gem.homepage = 'https://github.com/jruby/activerecord-jdbc-adapter'
-  gem.license = "BSD"
+  gem.license = 'BSD-2-Clause'
```

Comment:
Will anyone perceive this as a license change or was BSD by itself just too ambiguous?

Reply:
RG was hitting me with a warning that I need to pick smt, as I tested building the gemspec